plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/GcsStorageFactory.java

```diff
@@ -58,6 +58,7 @@ public GcsStorageFactory(HdfsEnvironment hdfsEnvironment, HiveGcsConfig hiveGcsC
             throws IOException
     {
         this.hdfsEnvironment = requireNonNull(hdfsEnvironment, "hdfsEnvironment is null");
+        hiveGcsConfig.validate();
```

Comment:
Should support for `validate` method in `Config` class be added to airlift?

Reply:
`@PostConstruct` exists?

Reply:
airlift supports standard bean validation, so i don't think so 

normally it works awesome
the problem for this particular config is with the `testExplicitPropertyMappings` that wants to set all properties to some value, and they are mutually exclusive